File: Algorithms/linearSearch.js
let arr = [22, 55, 88, 77, 44, 11, 66, 989, 45, 78];
let targetValue = 989;
for (let i = 1; i <= arr.length; i++) {
  if (arr[i] == targetValue) {
    console.log("The targetValue is located at index number " + i);
  }
}

File: Algorithms/linearSearch.c
#include <stdio.h>
int linearSearch(int *arr, int target, int size)
{
    for (int index = 0; index < size; index++)
    {
        if (arr[index] == target)
        {
            printf("The target value %d is present at %d index.", target, index);
        }
    }
}
int main()
{
    int arr[6] = {5, 4, 7, 98, 6, 4};
    int target = 98;
    int size = 6;
    linearSearch(&arr, target, size);
}
